scylla-cql/src/errors.rs

```diff
@@ -46,6 +46,10 @@ pub enum QueryError {
     #[error("Request timeout: {0}")]
     RequestTimeout(String),
 
+    /// No known node found to perform query
+    #[error("No known node found: {0}")]
+    NoKnownNodeFoundError(String),
```

Comment:
I'm not convinced by this name. What is `KnownNode` in this context? Maybe something like `EmptyQueryPlan`?